SwanCustomEnvironments/swancustomenvironments/scripts/makenv.sh

```diff
@@ -171,16 +183,28 @@ _log "ENV_NAME:${ENV_NAME}"
 # Make sure the Jupyter server finds the new environment kernel in /home/$USER/.local
 mkdir -p /home/$USER/.local/share/jupyter/kernels
 ln -f -s ${ENV_PATH}/share/jupyter/kernels/${ENV_NAME} /home/$USER/.local/share/jupyter/kernels/${ENV_NAME} | tee -a "${LOG_FILE}"
+# Avoid an ipykernel installation warning message
+export JUPYTER_PATH=${ENV_PATH}/share/jupyter:$JUPYTER_PATH
```

Comment:
What is this? This is not conda related because it is done for all the env types.

Reply:
Indeed. I realised this line is able to avoid that boring warning message saying the configured kernelspec wasn't found in a given list of paths.

Reply:
Ok, then this needs to go in another commit with its own message.

Reply:
Also, does the variable need to be exported?

Reply:
I believe so, in order for the system to actually find the kernelspec for the environment. But I'll test without exporting it and see if everything is ok.

Reply:
What is the line of this script exactly that needs this variable definition to be done? It would be good to have the definition of the variable right before the command that requires it.

Reply:
`python -m ipykernel install --name "${ENV_NAME}" --display-name "Python (${ENV_NAME})" --prefix "${ENV_PATH}"`
This is the line where ipykernel installs the environment kernel.
As you can see, the prefix is the path to the environment. And unlike python and accpy, conda environments are installed on `/opt/conda/envs` (something like this).

Reply:
Then I believe you could just do:
```
# JUPYTER_PATH is set to that the installation of ipykernel does not complain about non-found kernelspec
JUPYTER_PATH=${ENV_PATH}/share/jupyter python -m ipykernel install --name "${ENV_NAME}" --display-name "Python (${ENV_NAME})" --prefix "${ENV_PATH}"
```